packages/db/src/services/DatabaseService.ts

```
@@ -133,18 +137,232 @@ export class DatabaseService {
-      console.log("Inserting packet data into database");
-      const flattenedData = this.flattenTelemetryData(message);
-      await this.telemetryPacketRepo.save(flattenedData);
+  public async getDrivers() {
+    try {
+      const drivers = await this.driverRepo.find();
+      return drivers.map((driver) => ({
+        Rfid: driver.Rfid,
+        driver: driver.Name,
+      }));
+    } catch (error: unknown) {
+      console.error("Error getting drivers");
```

ESLint

[error] Unexpected console statement.

(no-console)